test/integration/webhook/v1alpha1/workload_test.go

```diff
@@ -50,13 +50,13 @@ var _ = ginkgo.Describe("Workload defaulting webhook", func() {
 			workload := testing.MakeWorkload(workloadName, ns.Name).Obj()
```

Comment:
But `MakeWorkload` already sets some defaults (in particular, the podset name). Maybe we should use `kueue.Workload` directly in this test.

Same for other API objects.